alfresco-health-processor-platform/src/main/amp/config/alfresco/subsystems/HealthProcessor/default/processing-context.xml

```diff
@@ -17,10 +17,16 @@
         <constructor-arg name="processorService" ref="eu.xenit.alfresco.healthprocessor.processing.ProcessorService"/>
     </bean>
 
+    <bean id="eu.xenit.alfresco.healthprocessor.util.AlfrescoTransactionHelper"
+            class="eu.xenit.alfresco.healthprocessor.util.AlfrescoTransactionHelper">
+        <constructor-arg name="retryingTransactionHelper" ref="retryingTransactionHelper"/>
+    </bean>
+
     <bean id="eu.xenit.alfresco.healthprocessor.processing.ProcessorService"
-            class="eu.xenit.alfresco.healthprocessor.processing.ProcessorService">
+            class="eu.xenit.alfresco.healthprocessor.processing.ProcessorService" autowire="constructor">
```

Comment:
I thought just specifying ctor args was sufficient? :thinking:
Why do you need to specify the `autowire` argument?

Reply:
Because I want to autowire a `List<HealthProcessorPlugin>` and `List<HealthProcessorReporter>`. Notice that those two ctor arguments are not explicitly specified here. 

Explicit specifying (e.g. `ref="..."`) overwrites the `autowire="constructor"` config for that specific ctor argument.